packages/js/src/classic-editor.js

```diff
@@ -47,5 +56,20 @@ domReady( async () => {
 	// Responsibility:
 	// - render metabox
 	// - provide slot/fill mechanism
-	initClassicEditorIntegration( {} );
+	// Expose registerReactComponent as an alternative to registerPlugin.
+	registerApis( [ { _registerReactComponent: registerReactComponent } ] );
+
+	renderReactRoot( {
+		target: "wpseo-metabox-root",
+		children: (
+			<SeoProvider>
+				<MetaboxSlot />
+				<MetaboxFill>
+					<Metabox />
+				</MetaboxFill>
+			</SeoProvider>
+		),
+		theme: { isRtl: Boolean( wpseoScriptData.metabox.isRtl ) },
+		location: "metabox",
+	} );
```

Comment:
To keep the root script clean, this could live in its own file in the end. But fine for now?